src/main/kotlin/net/atos/zac/mail/model/EMails.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.mail.model
+
+import jakarta.json.bind.annotation.JsonbProperty
+
+class EMails(@field:JsonbProperty("Messages") private var eMails: List<EMail>) {
+    fun geteMails(): List<EMail> {
```

Comment:
If we convert this to a data class you get the getters and setters from free.

Reply:
Seems we rely on the setter/getter names for JSONB serialization. Data class messes up with this :(